docs/source/tutorials/own-data.rst

```diff
@@ -73,29 +73,16 @@ Once succeeded, the command will create a set of folders and files under the ``/
 
 The input configuration JSON
 ```````````````````````````````
-GraphStorm's graph construction tool relies on the configuration JSON to provide graph information. Explanations of the format of the configuration JSON contents could be found in the :ref:`GraphStorm Graph Configuration JSON <gconstruction-json>`. Below show the contents of the examplary ACM `config.json` file.
+The above command automatically creates the examplary ACM `config.json` file, some of which are listed below.
 
-.. code-block:: json
+.. code-block:: yaml
```

Comment:
Isn't it a json file?

Reply:
Yes, it is a json. But in rst files, the rst interpreter can render yaml better for JSON contents. So I change it to yaml.